scripts/deployment/phase2-assets/collaterals/deploy_lido_wsteth_collateral.ts

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs'
+import hre, { ethers } from 'hardhat'
+import { getChainId } from '../../../../common/blockchain-utils'
+import { networkConfig } from '../../../../common/configuration'
+import { bn, fp } from '../../../../common/numbers'
+import {
+  getDeploymentFile,
+  getAssetCollDeploymentFilename,
+  IAssetCollDeployments,
+  getDeploymentFilename,
+  fileExists,
+} from '../../common'
+import { priceTimeout, oracleTimeout } from '../../utils'
+import { LidoStakedEthCollateral } from '../../../../typechain'
+import { ContractFactory } from 'ethers'
+
+async function main() {
+    // ==== Read Configuration ====
+    const [deployer] = await hre.ethers.getSigners()
+
+    const chainId = await getChainId(hre)
+
+    console.log(`Deploying Collateral to network ${hre.network.name} (${chainId})
+    with burner account: ${deployer.address}`)
+
+    if (!networkConfig[chainId]) {
+        throw new Error(`Missing network configuration for ${hre.network.name}`)
+    }
+
+    // Get phase1 deployment
+    const phase1File = getDeploymentFilename(chainId)
+    if (!fileExists(phase1File)) {
+        throw new Error(`${phase1File} doesn't exist yet. Run phase 1`)
+    }
+    // Check previous step completed
+    const assetCollDeploymentFilename = getAssetCollDeploymentFilename(chainId)
+    const assetCollDeployments = <IAssetCollDeployments>getDeploymentFile(assetCollDeploymentFilename)
+
+    // Get Oracle Lib address if previously deployed (can override with arbitrary address)
+    const deployedCollateral: string[] = []
+
+    /********  Deploy Lido Staked ETH Collateral - wstETH  **************************/
+
+    const LidoStakedEthCollateralFactory: ContractFactory = await hre.ethers.getContractFactory(
+      'LidoStakedEthCollateral'
+    )
+
+    const collateral = <LidoStakedEthCollateral>await LidoStakedEthCollateralFactory.connect(deployer).deploy(
+      {
+        priceTimeout: priceTimeout.toString(),
+        chainlinkFeed: networkConfig[chainId].chainlinkFeeds.ETH,
+        oracleError: fp('0.005').toString(), // 0.5%,
+        erc20: networkConfig[chainId].tokens.wstETH,
+        maxTradeVolume: fp('1e3').toString(), // 1k $ETH,
```

Comment:
Surely we don't want to cap trades involving wstETH to $1k...we've used $1m everywhere else.

Reply:
yeah i think when i changed this i was translating "1k eth @ $1k is $1m" which is something i've had to do in other protocols... will revert